packages/angular_devkit/build_angular/src/builders/karma/schema.json

```diff
@@ -143,6 +143,14 @@
       "default": ["**/*.spec.ts"],
       "description": "Globs of files to include, relative to workspace or project root. \nThere are 2 special cases:\n - when a path to directory is provided, all spec files ending \".spec.@(ts|tsx)\" will be included\n - when a path to a file is provided, and a matching spec file exists it will be included instead."
     },
+    "exclude": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "default": [],
+      "description": "Globs of files to exclude, relative to workspace or project root."
```

Comment:
Wouldn't this always be relative to the project source root?

Reply:
Yes. I’ll update the `include` description as well.